3p/integration.js

```diff
@@ -360,8 +361,8 @@ function triggerResizeRequest(width, height) {
   nonSensitiveDataPostMessage('embed-size', {width, height});
 }
 
-function triggerRenderStart() {
-  nonSensitiveDataPostMessage('render-start');
+function triggerRenderStart(width, height) {
```

Comment:
Let's make the param a `opt_data` object, for future extensibility.
A JS doc for this optional param would help.

We should also update `ads/README.md`.